### PR TITLE
feat: use rewired/assemble brand palette

### LINKED ISSUE #1407: Use Rewired branding
**Is your feature request related to a problem? Please describe.**

There is nothing visually distinctive about the Spoke Rewired fork of Spoke.

**Describe the solution you'd like**

Use Rewired branding for Spoke Rewired.

**Describe alternatives you've considered**

Making no changes.

**Additional context**

N/A

### PR DESCRIPTION
**Pull request by bchrobot**
## Description

Switch to Rewired brand palette.

This includes a few MUI updates for particularly difficult to resolve components. Specifically:

- Tabs: `<TabPanel>` component gets pulled up
- Campaign builder: cards and standardizing on `<SectionWrapper>` component. Cards changes are minimal and meant to be superseded by work started in  https://github.com/politics-rewired/Spoke/tree/chore/cards-mui4

## Motivation and Context

Coherent visual identity.

Closes #1407.

## How Has This Been Tested?

This has been tested locally.

Changes can be previewed here: https://stagingenv.spokerewired.com/

## Screenshots (if appropriate):

_Note: these were taken before swapping font out for Karla_

<table>

<tr><td>

Screen Shot 2022-08-30 at 2 51 27 PM

<p>Log in</p>

</td><td>

Screen Shot 2022-08-30 at 8 30 38 PM

<p>Org selection</p>

</td></tr>

<tr><td>

Screen Shot 2022-08-30 at 2 53 23 PM

<p>Campaign List</p>

</td><td>

Screen Shot 2022-08-30 at 2 53 51 PM

<p>Campaign Template List</p>

</td></tr>

<tr><td>

Screen Shot 2022-08-30 at 2 54 08 PM

<p>Campaign Groups</p>

</td><td>

Screen Shot 2022-08-30 at 2 54 25 PM

<p>People</p>

</td></tr>

<tr><td>

Screen Shot 2022-08-30 at 2 54 36 PM

<p>Teams and Team Editor</p>

</td><td>

Screen Shot 2022-08-30 at 2 54 43 PM

<p>Assignment Control</p>

</td></tr>

<tr><td>

Screen Shot 2022-08-30 at 2 54 50 PM

Autosending

</td><td>

Screen Shot 2022-08-30 at 2 55 18 PM

Tags

</td></tr>

<tr><td>

Screen Shot 2022-08-30 at 2 55 45 PM

Message Review

</td><td>

Screen Shot 2022-08-30 at 8 37 12 PM

Conversation View

</td></tr>

<tr><td>

Screen Shot 2022-08-30 at 2 55 58 PM

Bulk Script Editor

</td><td>

Screen Shot 2022-08-30 at 3 00 39 PM

Assignment Requests

</td></tr>

<tr><td>

Screen Shot 2022-08-30 at 8 59 57 PM

Troll Alarms

</td><td>

Screen Shot 2022-08-30 at 3 00 54 PM

Integrations

</td></tr>

<tr><td>

Screen Shot 2022-08-30 at 8 40 01 PM

Settings - General

</td><td>

Screen Shot 2022-08-30 at 8 40 24 PM

Settings - Trollbot

</td></tr>

<tr><td>

Screen Shot 2022-08-30 at 8 41 01 PM

Texter Assignments

</td><td>

Screen Shot 2022-08-30 at 8 41 23 PM

Texter Replies

</td></tr>

<tr><td>

Screen Shot 2022-08-30 at 8 42 09 PM

User Edit

</td><td>

Screen Shot 2022-08-30 at 8 42 58 PM

Login

</td></tr>

<tr><td>

Screen Shot 2022-08-30 at 8 43 43 PM

Campaign Operations

</td><td>

Screen Shot 2022-08-30 at 8 45 52 PM

Campaign Details

</td></tr>

<tr><td>

Screen Shot 2022-08-30 at 8 46 33 PM

Campaign Builder

</td><td>

Screen Shot 2022-08-30 at 8 47 18 PM

Interaction Steps

</td></tr>

<tr><td>

Screen Shot 2022-08-30 at 8 47 35 PM

Script Builder

</td><td>

Screen Shot 2022-08-30 at 8 48 32 PM

Texters

</td></tr>

<tr><td>

Screen Shot 2022-08-30 at 8 48 57 PM

Contacts - CSV

</td><td>

Screen Shot 2022-08-30 at 8 50 32 PM

Contacts - VAN

</td></tr>

<tr><td>

Screen Shot 2022-08-30 at 8 53 32 PM

Task Working

</td><td>

Screen Shot 2022-08-30 at 8 57 04 PM

VAN Mappings

</td></tr>

</table>

## Documentation Changes

N/A

## Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have included updates for the documentation accordingly.

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1202945643298614

### CODE REVIEW

**Comment by ajohn25** [comment]
@bchrobot only flag at a quick glance is for the campaign builder. See below for the white/grey only and with no `<Avatar>` gray circles suggestions from Asana. I think possibly same idea for the texter "send first messages" etc. buttons to also stay white but defer to @kennyycheng!
image

**Comment by kennyycheng** [comment]
> 

@ajohn25 Oh yeah, good call! I have a little handoff spec if that helps out here. It has the avatar circles (just a little lighter though?), but I don't have strong feelings about keeping it or removing it, so long as the icon+color still feel apparent.

frame_195

Otherwise, I think the rest of the palette looks good to me @bchrobot!

File: src/containers/AdminExternalSystems.tsx [hunk 1]
```diff
@@ -6,6 +6,7 @@ import DialogActions from "@material-ui/core/DialogActions";
 import DialogContent from "@material-ui/core/DialogContent";
 import DialogContentText from "@material-ui/core/DialogContentText";
 import DialogTitle from "@material-ui/core/DialogTitle";
+import Fab from "@material-ui/core/Fab";
```
Comment (ajohn25) [inline src/containers/AdminExternalSystems.tsx:9]: The `<Fab>` upgrades here should be across the board so could either disable the v0 imports and close #1381 or check off that first checkbox and do a quick separate PR for disabling imports!

File: src/containers/AdminCampaignEdit/index.jsx [hunk 2]
```diff
@@ -905,107 +895,31 @@ class AdminCampaignEdit extends React.Component {
...
-          } = this.sectionSaveStatus(section);
+          const { sectionIsSaving, relatedJob, jobId } = this.sectionSaveStatus(
+            section
+          );
           const sectionCanExpandOrCollapse =
             (section.expandAfterCampaignStarts ||
               !this.props.campaignData.campaign.isStarted) &&
             (isAdmin || section.expandableBySuperVolunteers);
 
-          if (sectionIsSaving) {
```
Comment (ajohn25) [inline src/containers/AdminCampaignEdit/index.jsx:931]: gotta love all the red here 😁

**Review by ajohn25 — APPROVED** [review]
🔥

**Comment by hiemanshu** [comment]
Sorry for taking a bit to get into this @bchrobot and @politics-rewired/organizing. Here's the results from my contrast tests. A bunch of places we need to work on and get alternative colors. To quickly explain readable results, AA stands for Ideal Support with Normal Text size and Large Text Size (Large Text needs a lower contrast ratio to pass), AAA stands for Specialized Support, something we can choose to not worry about for a little bit. AA is the recommended minimum to pass. 

## Passed:
Screenshot 2022-09-08 at 7 47 23 PM
Screenshot 2022-09-08 at 7 47 49 PM
Screenshot 2022-09-08 at 7 49 46 PM
Screenshot 2022-09-08 at 7 51 35 PM
Screenshot 2022-09-08 at 7 51 52 PM
Screenshot 2022-09-08 at 7 54 54 PM
Screenshot 2022-09-08 at 7 35 42 PM
Screenshot 2022-09-08 at 7 36 17 PM
Screenshot 2022-09-08 at 7 37 39 PM
Screenshot 2022-09-08 at 7 41 19 PM

## Failed: 
### Script Editor Custom Fields
Screenshot 2022-09-08 at 7 44 57 PM

### Optout Button
Screenshot 2022-09-08 at 7 50 15 PM

### Campaign Name (Campaign List)
Screenshot 2022-09-08 at 7 36 54 PM

### Completed Campaign Name (Campaign List)
Screenshot 2022-09-08 at 7 38 36 PM

### Details Page Overdue Date
Screenshot 2022-09-08 at 7 39 27 PM

### Uncompleted Sections Campaign Editor
Screenshot 2022-09-08 at 7 43 20 PM

**Review by hiemanshu — CHANGES_REQUESTED** [review]
Overall looks good! A few places has hard coded colors, those should be replaced. And we should talk about the color contrasts that are failing.

**Comment by hiemanshu** [comment]
From the failed ones, the most commonly seen ones/ones that need immediate attention would be:

Script Editor Custom Fields
Campaign Name (Campaign List)
Completed Campaign Name (Campaign List)

**Review by hiemanshu — APPROVED** [review]

**Pull request merged**